store/migrate_test.go

```diff
@@ -217,6 +217,51 @@ func (d *DBV3) compare(td testdb) bool {
 	return true
 }
 
+type DBV4 struct {
```

Comment:
Thanks I was wrestling with migration tests. I got a better idea of the problem now.
Since there were no V4 or V5, I thought there was a version offset and V3 corresponded to V5 in schema